### DISCUSSION TITLE
C++ Intellisense snippet language is not consistant

### DISCUSSION THREAD

**Discussion by KNOXZhao** [microsoft/vscode-cpptools#10645] [Q&A]
I am seeking assistance with an issue I encountered while running VSCode on my Windows 10 Family (Chinese) operating system. Previously, I experienced no problems, however, today while working on my C++ program, I discovered that the language displayed on the error snippet is in Chinese, while other functions are displayed in English as usual. To clarify, the Simplified Chinese pack was not installed, and the default language for VSCode is English. I also verified that "locale" is set to "en" in argv.json. Despite searching for solutions to this problem, I have not found any helpful suggestions. Attached is a screenshot of the issue.
<img width="866" alt="Screenshot" src="https://user-images.githubusercontent.com/90520126/223793660-30ad9dc4-7b9f-4055-a00c-bec867c26184.png">

**Comment by bobbrow** [comment]
Our extension has the localization info embedded into it.  It does not use the Simplified Chinese pack extension.  Unless you explicitly select English as the language, VS Code will tell us to use the Chinese strings instead because you're in using the Chinese OS.  Try running the "Configure Display Language" command in VS Code to select English and then reload the window.

**Reply by Colengms** [reply]
@KNOXZhao .  The C/C++ Extension implements localization using the `vscode-nls` and `vscode-nls-dev` modules. Through those modules, VS Code itself has control over which strings are selected by the extension.  You might consider opening an issue against VS Code, as the language used by VS Code itself should be the same language those modules select within the extension.

**Reply by sean-mcmanus** [reply]
@bobbrow @Colengms Have you seen https://github.com/microsoft/vscode-cpptools/issues/10632 ? The user might be hitting that bug, which only repros with a non-English OS.

**Reply by KNOXZhao** [reply]
Thanks for the reply, but I tried configuring the display language to en, and the problem still exists. I also tried reinstalling and uninstalling the Chinese language package and that also won't work.

**Reply by KNOXZhao** [reply]
I think I encounter the same problem with #10632. We share similar system settings and I perform the same methods to tackle the problem but I still could not fix that.
